API/http.go

```diff
@@ -63,11 +63,13 @@ func main() {
 	putRouter.Handle("/api/accounts/patients/{id:[0-9]+}/dosages/scheduled", handlerWrapper(addScheduledDosages))
 	putRouter.Handle("/api/accounts/patients/{id:[0-9]+}/notes", handlerWrapper(addNote))
 	putRouter.Handle("/api/general/videos", handlerWrapper(addVideo))
+	putRouter.Handle("/api/admin/addMedicine", handlerWrapper(pushMedicine))
 
 	// DELETE Requests for Deleting
 	deleteRouter := router.Methods("DELETE").Subrouter()
 	deleteRouter.Handle("/api/accounts/patients/{id:[0-9]+}", handlerWrapper(deletePatient))
 	deleteRouter.Handle("/api/accounts/physicians/{id:[0-9]+}", handlerWrapper(deletePhysician))
+	deleteRouter.Handle("/api/admin/deleteMedicine/{id:[0-9]+}", handlerWrapper(deleteMedicine))
```

Comment:
I think we should have proper url names for this, not `deleteMedicine` and `addMedicine`

Reply:
like?

Reply:
something like `api/ ... /medicine/{id}`.

I am not sure what to put at the dots though. Maybe admin, as you have it, since I cannot think of anything else :p

Reply:
oh alright I get it